build.gradle

```diff
@@ -63,12 +66,18 @@ publishing {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8",
             "org.web3j:core:$web3jVersion",
+            "org.web3j:web3j-evm:$web3jVersion",
             "org.testcontainers:testcontainers:$testContainersVersion",
             "org.junit.jupiter:junit-jupiter-engine:$junitVersion",
             "org.junit.jupiter:junit-jupiter-api:$junitVersion",
             "org.junit.jupiter:junit-jupiter-params:$junitVersion",
             "io.github.microutils:kotlin-logging:$kotlinLoggingVersion",
-            "ch.qos.logback:logback-classic:$logbackVersion"
+            "ch.qos.logback:logback-classic:$logbackVersion",
+            "org.apache.logging.log4j:log4j-api:$log4jVersion",
+            "org.apache.logging.log4j:log4j-core:$log4jVersion",
+            "com.google.guava:guava:$guavaVersion",
```

Comment:
do we need all these dependencies?

Reply:
Leftovers from before the evm was published (now part of pom file), will remove.